mqtt-connect/websocket/src/main/java/com/cumulocity/mqtt/connect/client/websocket/MqttWebSocketClient.java

```diff
@@ -54,11 +68,17 @@ public void onMessage(String message) {
 
     @Override
     public void onClose(int code, String reason, boolean remote) {
+        if (!isOpen()) {
+            connectionError = String.format("Cannot connect: %s %s", reason, code);
```

Comment:
super.isOpen would be cleaner. Same in onError()

Reply:
Changed